account_banking_mandate/models/account_banking_mandate.py

```diff
@@ -177,8 +177,8 @@ def _check_valid_state(self):
 
     @api.model
```

Comment:
Please take the occasion to convert this to multi-record (`@api.model_create_multi`)